src/module-init/module-init.c

```diff
@@ -135,6 +136,14 @@ void RS_moduleInfoFunc(RedisModuleInfoCtx *ctx, int for_crash_report) {
   RedisModule_InfoAddFieldDouble(ctx, "total_cycles", stats.totalCycles);
   RedisModule_InfoAddFieldDouble(ctx, "total_ms_run", stats.totalTime);
 
+  // Errors statistics
+  RedisModule_InfoAddSection(ctx, "errors");
+  IndexingErrorsStats indexing_errors = total_info.indexing_errors;
+    // indexing errors
+  RedisModule_InfoAddFieldDouble(ctx, "errors_indexing_failures", indexing_errors.indexing_failures);
+  // highest number of failures out of all specs
+  RedisModule_InfoAddFieldDouble(ctx, "errors_indexing_failures_max", indexing_errors.max_indexing_failures);
```

Comment:
what is the real world purpose for this field? also consider rename to `max_indexing_errors` or something similar